connectors/tests/ftest.sh

```diff
@@ -47,9 +47,9 @@ if [[ $PERF8 == "yes" ]]
 then
     if [[ $PLATFORM == "darwin" ]]
     then
-      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil -c $ELASTIC_INGEST --debug & PID=$!
+      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil --psutil-max-rss 209715200 -c $ELASTIC_INGEST --debug & PID=$!
```

Comment:
Could we add a comment, that 209715200 bytes are 200MiB?

Reply:
I think I'll do better : allow `--psutil-max-rss 200MiB`